react/__tests__/index.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('Hello tests', () => {
+  it('should be a success', () => {
+    expect(true).toBe(true)
```

Comment:
Useful?

Reply:
I would leave this here. It is not that useful now, but it is kind of a reminder that tests should be here... also, the config is already in place, so this makes it easier to start adding tests.

Reply:
I'd keep the code base clean by removing this file and creating an issue in the repo to add tests or by adding a real first test. But that's my opinion.

Reply:
I agree with @victorhmp, It's a good starting pack for future development.